docs/block-node/block-node-overview.md

```diff
@@ -0,0 +1,138 @@
+
+Block Nodes distribute blocks to downstream clients—including Mirror Nodes, other Block Nodes, and applications—so anyone can access real-time or historical data. They also generate cryptographic proofs for transactions and state, making it possible for users and applications to independently verify the accuracy of the blockchain’s history without relying on a single provider.
+
+Finally, Block Nodes help the network scale and run smoothly. When a Consensus Node needs to catch up with its peers or recover from downtime, it can rely on Block Nodes to provide the latest data and state snapshots—relieving Consensus Nodes from heavy data storage duties and improving overall efficiency.
+
+## Core Block Node Services
+
+Block Nodes provide several core services that turn block streams into reliable, consumable data for the network:
+
+- Block stream ingestion, verification, and distribution.
+- Cryptographic proofs for transactions and network state.
+- Durable storage of blocks, sidecars, and consensus state.
+- Real-time and historical data streaming to downstream clients.
+- Random-access retrieval of blocks and state at specific block heights.
+- State snapshots and reconnect services to help Consensus Nodes catch up or recover.
+
+## Example Use Cases
+
+- Running an API server for wallets, explorers, or dApps to access verified blockchain data.
+- Operating analytics or aggregation pipelines using live and historical block streams.
+- Providing specialized compliance, archival, or network recovery support for Hedera services.
+
+## Block Node Types
+
+|    **Type**     |                         **Description**                          |        **Typical Operators**        |                              **Key Focus**                              |
+|-----------------|------------------------------------------------------------------|-------------------------------------|-------------------------------------------------------------------------|
+| Tier 1          | Receive streams directly from Consensus Nodes; high reliability. | Governing Council, trusted entities | Verification, reconnect, state snapshots .                              |
+| Tier 2          | Receive from Tier 1/Tier 2; permissionless.                      | Community, enterprises              | Streaming, proofs, geographic redundancy.                               |
+| Full-History    | Retain genesis-to-present history in hot/online storage.         | Any                                 | Random-access queries, full historical streams.                         |
+| Partial-History | Store only recent blocks (e.g., 30-90 days).                     | Cost-optimized                      | Recent-data streaming, light storage footprint.                         |
```

Comment:
Partial history is more of a retention policy result under LFH. I wouldn't call it out as a general Block Node type